### ISSUE TITLE
Fix set_fast_options() for R CMD check

### ISSUE THREAD

**Issue by appelmar** [bfast2/bfast#73]
R CMD check gives 

```
> set_fast_options()
Error in as.environment("package:strucchange") : 
  no item called "package:strucchange" on the search list
Calls: set_fast_options
```

**Comment by appelmar** [comment]
Let's discuss later. As it is, it only solves the examples and `set_fast_options` only works if users explicitly call `library(strucchangeRcpp)` before. To solve this, we could add the `strucchangeRcpp` package  as `Depends` instead of `Imports`.

**Comment by GreatEmerald** [comment]
Oh, never mind, this is the issue, I thought this was the pull request!

**Comment by GreatEmerald** [comment]
Fixed in #78 by putting `strucchangeRcpp` into `Depends`.
